lib/cdo/rack/attack.rb

```diff
@@ -0,0 +1,59 @@
+require 'rack/attack'
+require 'active_support/core_ext/numeric/time'
+
+class Rack::Attack
+  # maximum requests per minute over a 4-minute stretch.
+  MAX_TABLE_READS_PER_MIN = 2 * 60
```

Comment:
I think we will likely want to make these dynamic as well; e.g. put them in DCDO and have an action that causes the limits to be updated.